src/Cupboard/Resources/Windows/Winget/WingetPackage.cs

```diff
@@ -6,6 +6,8 @@ public sealed class WingetPackage : Resource
     {
         public string Package { get; set; }
         public PackageState Ensure { get; set; }
+        public bool? IgnoreChecksum { get; set; }
```

Comment:
The boolean should not be nullable.

Reply:
Since this correlates with the argument `--force`, perhaps this should be called `Force` instead?

Reply:
`--force` just means to ignore checksum. I was trying to get some consistency with Chocolatey flags. (Also `IgnoreChecksum` is more self explanatory than `Force`)

Reply:
I understand your point, but I would prefer if we use the Winget terminology to keep things in sync.

Reply:
Ok. Will change